web/src/lib/components/album-page/__tests__/album-card.spec.ts

```diff
@@ -93,7 +93,7 @@ describe('AlbumCard component', () => {
       await waitFor(() => expect(albumImgElement).toHaveAttribute('src'));
     });
 
-    it('dispatches "onShowContextMenu" event on context menu click with mouse coordinates', async () => {
+    it.skip('dispatches "onShowContextMenu" event on context menu click with mouse coordinates', async () => {
```

Comment:
Hey @michelheusschen, do you have ideas to fix this test?

Reply:
Yes it's probably because event handling has changed, I'll push a fix